client/src/components/new-request/search.vue

```diff
@@ -1,89 +1,161 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal">
-    <v-row justify="end">
-      <v-col cols="12" class="copy-normal">I need a name to:</v-col>
+    <v-row cols="12">
+      <v-col cols="6" class="font-weight-bold">I need a name to:</v-col>
```

Comment:
Do you need "copy-normal" style here as well?

Reply:
Yessir, should be all good as it is inherited.